Sources/QuaternionModule/Quaternion.swift

```diff
@@ -350,6 +413,24 @@ extension Quaternion: Hashable {
     return lhs.components == rhs.components
   }
 
+  /// Transformation equality comparison
+  ///
+  /// Returns a Boolean value indicating whether the 3D transformations of this
+  /// quaternion equals the 3D transformation of `other`.
+  ///
+  /// - Parameter other: The value to compare.
+  /// - Returns: True if the transformation of this quaternion equals `other`.
+  @_transparent
+  public func transformEquals(_ other: Quaternion) -> Bool {
```

Comment:
How do we feel about how this reads at a use site?
```
if q.transformEquals(p) { /* ... */ }
```
I don't love that "transform" may read as a verb if you don't know what you're looking at. What if we used something like this instead:
```
if q.equals(as3DTransform: p) { /* ... */ }
```
I'm not totally set on this either way, but let's give it some thought.

Reply:
> How do we feel about how this reads at a use site?
> 
> ```
> if q.transformEquals(p) { /* ... */ }
> ```

Yeah, you are right. This can easily be misread. We could maybe use `transformationEquals` to make it more explicit.

> ```
> if q.equals(as3DTransform: p) { /* ... */ }
> ```

I think, I do prefer this spelling over `transformEquals`/`transformationEquals`. If we want to (somewhat) reuse existing naming patterns, we could utilize `append(contentsOf:)` and `insert(contentsOf:at:)` on `Array` and use:

```swift
if q.equals(transformationOf: p) { /* ... */ }
```

Though, I think it is a little less obvious and does not mention 3D explicitly.

Reply:
Let's think about this over the weekend and see if we can make a provisional decision on Monday.

Reply:
Any further thoughts on this? If not, let's go with `equals(as3DTransform:)` for now?

Reply:
Perhaps `q.isSame3DTransform(as: p)` or some such?

Reply:
I changed it to `equals(as3DTransform:)` for now. I think, it reads a little nicer at the call site.